content/08.transmission.md

```diff
@@ -136,57 +136,27 @@ The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@ht
+In addition to Black Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic Americans and in Native American and Alaskan Native communities, including the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055].
+In Brazil indigenous communities likewise carry an increased burden of COVID-19 [@doi:10.1126/science.abc0073].
+In the United Kingdom, nonwhite ethnicity (principally Black or South Asian) was one of several factors found to be associated with a higher risk of death from COVID-19 [@doi:10.1038/s41586-020-2521-4].
+From a genetic standpoint, it is highly unlikely that ancestry itself predisposes individuals to contracting COVID-19 or to experiencing severe COVID-19 outcomes.
+Examining human genetic diversity indicates variation over a geographic continuum, and that most human genetic variation is associated with the African continent [@doi:10.1038/ng1438].
+African-Americans are also a more genetically divere group relative to European-Americans, with a large number of rare alleles and a much smaller fraction of common alleles identified in African-Americans [@doi:10.1146/annurev.genom.9.081307.164258].
+Therefore, the idea that African ancestry might convey some sort of genetic risk for severe COVID-19 seems unlikely given what is known about worldwide human genetic diversity [@doi:10.1126/science.abd4842], with some minor exceptions, such as that it's possible variants in the major histocompatability complex (MHC) could vary regionally and possibly convey some protective effect [@doi:10.1016/j.ijid.2020.07.016].
+However, such variants are unlikely to explain the patterns that are appearing on a global scale.
+Reports from several sub-Saharan African countries have indicated that the effects of the COVID-19 pandemic have been less severe than expected based on the outbreaks in China and Italy.
+In Kenya, for example, estimates of national prevalence based on testing blood donors for SARS-CoV-2 antibodies were consistent with 5% of Kenyan adults having recovered from COVID-19 [@doi:10.1101/2020.07.27.20162693].
+This high seroprevalence of antibodies lies in sharp contrast to the low number of COVID-19 fatalities in Kenya, which at the time was 71 out of 2093 known cases [@doi:10.1101/2020.07.27.20162693].
+Likewise, a serosurvey of health care workers in Blantyre City, Malawi reported an adjusted antibody prevalence of 12.3%, suggesting that the virus had been circulating more widely than thought and that the death rate was up eight times lower than models had predicted [@doi:10.1101/2020.07.30.20164970].
```

Comment:
My issue with this preprint is that they assume that 12.3% rate measured in healthcare workers extrapolates to the whole population. So their 8x estimate they give is likely a bit too rosy. I don't know if the phrasing "up to" does a sufficient job of providing a bit of skepticism about this number without undermining their interesting findings.